certified-connectors/VEV-IQ/apiDefinition.swagger.json

```diff
@@ -0,0 +1,559 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "VEV-IQ",
+    "description": "VEV-IQ custom connectors enables you to retrieve data from your charging infrastructure using APIs and integrate it into Microsoft tools.",
```

Comment:
Please don't use API keywork in title and description

Reply:
Done! thank you